src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowRun/sidepanel.jelly

```diff
@@ -34,6 +34,37 @@
             <p:console-link/>
             <l:task icon="images/24x24/notepad.png" href="${buildUrl.baseUrl}/configure" title="${h.hasPermission(it,it.UPDATE)?'%Edit Build Information':'%View Build Information'}"/>
             <st:include page="delete.jelly"/>
+            <j:if test="${it.isBuilding() and h.hasPermission(it,it.CANCEL)}">
+                <j:choose>
+                    <j:when test="${it.hasStopBeenCalled() == false}">
+                        <l:task icon="images/24x24/edit-delete.png"
+                                href="${buildUrl.baseUrl}/stop"
+                                title="${%Stop Build}"
```

Comment:
I'm open to alternative locations for this, but it definitely has to be more than just in the log. I suppose we could just transform the stop button's actions on the fly?